www/js/modules/charts/controllers/HistogramChartController.js

```diff
@@ -199,19 +199,25 @@ export default class HistogramChartController {
 
         let that = this;
 
+		// must remove histogram to update histogram
+	    d3.select('#myHistogram123').remove();
+		this.$histogram = null;
```

Comment:
you don't need the id because $histogram is the dom element already. You should just be able to do `this.$histogram.remove()`